packages/validate/src/lib/validation.ts

```diff
@@ -24,6 +25,10 @@ function formatPath(path: ReadonlyArray<number | string>): string | null {
 			formattedPath += `.${item}`
 		}
 	}
+
+	// N.B. We don't want id's in the path because they make grouping in Sentry tough.
+	formattedPath = formattedPath.replace(/id = [^,]+, /, '').replace(/id = [^)]+/, '')
```

Comment:
is it possible to do this in the sentry reporter rather than in the validation library? these are often useful to find in the console when debugging

Alternatively, we could remove the `T.model` type from validators. All it does is add this ID information, so if we're going to strip it out later we might as well remove that type entirely.

Reply:
I've been looking at the Sentry grouping mechanisms and it doesn't have, like, a regex system to ignore this bit and group on the other parts unfortunately :-/

I definitely could see this being useful in development — maybe it's something that is not stripped when `NODE_ENV === 'development'`?

Hm, the `T.Model` afaict has more than just id - it has things like `type = 'arrow'` and that sort of thing.